pkg/interceptors/interceptors.go

```diff
@@ -94,35 +95,6 @@ func GetInterceptorParams(i *triggersv1.EventInterceptor) map[string]interface{}
 			ip["objectRef"] = i.Webhook.ObjectRef
 			ip["header"] = i.Webhook.Header
 		}
-	case i.DeprecatedGitHub != nil:
-		if i.DeprecatedGitHub.EventTypes != nil {
-			ip["eventTypes"] = i.DeprecatedGitHub.EventTypes
-		}
-		if i.DeprecatedGitHub.SecretRef != nil {
-			ip["secretRef"] = i.DeprecatedGitHub.SecretRef
-		}
-	case i.DeprecatedGitLab != nil:
-		if i.DeprecatedGitLab.EventTypes != nil {
-			ip["eventTypes"] = i.DeprecatedGitLab.EventTypes
-		}
-		if i.DeprecatedGitLab.SecretRef != nil {
-			ip["secretRef"] = i.DeprecatedGitLab.SecretRef
-		}
-	case i.DeprecatedCEL != nil:
-		if i.DeprecatedCEL.Filter != "" {
-			ip["filter"] = i.DeprecatedCEL.Filter
-		}
-
-		if i.DeprecatedCEL.Overlays != nil {
-			ip["overlays"] = i.DeprecatedCEL.Overlays
-		}
-	case i.DeprecatedBitbucket != nil:
-		if i.DeprecatedBitbucket.EventTypes != nil {
-			ip["eventTypes"] = i.DeprecatedBitbucket.EventTypes
-		}
-		if i.DeprecatedBitbucket.SecretRef != nil {
-			ip["secretRef"] = i.DeprecatedBitbucket.SecretRef
-		}
```

Comment:
Is this a breaking change? is it possible to break this out into its own PR for visibility, or does it need to be coupled with the alpha->beta change?

Reply:
It should backwards compatible. Users can still create ELs with the old Deprecated interceptor syntax in the v1alpha1 API group. The defaulting webhook should convert it to the new syntax and the sink code can still work -- this defaulting logic has been around for a couple of releases so as long as the EL was created after v0.13 or after this change should be backwards compatible.